dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -534,20 +532,19 @@ public GHRepository getRepository(String repositoryId) {
     // The documentation doesn't list the possibilities https://github-api.kohsuke.org/apidocs/org/kohsuke/github/GHRef.GHObject.html#getType(),
     // but I'll assume it mirrors the 4 Git types: blobs, trees, commits, and tags.
     private String getCommitSHA(GHRef ref, GHRepository repository, String refName) throws IOException {
-        String sha = null;
+        String sha = ref.getObject().getSha();
         if ("commit".equals(ref.getObject().getType())) {
             sha = ref.getObject().getSha();
         } else if (ref.getObject().getType().equals("tag")) {
-            GHTagObject tagObject = repository.getTagObject(sha);
-            sha = tagObject.getObject().getSha();
+            sha = repository.getTagObject(ref.getObject().getSha()).getObject().getSha();
         } else if (ref.getObject().getType().equals("branch")) {
             GHBranch branch = repository.getBranch(refName);
             sha = branch.getSHA1();
         } else {
             // I'm not sure when this would happen.
             LOG.error("Unsupported GitHub reference object. Unable to find commit ID for type: " + ref.getObject().getType());
             // This is probably wrong, but we should mimic the behaviour from before since this is a hotfix.
-            sha = ref.getObject().getSha();
+            // sha = ref.getObject().getSha();
```

Comment:
Channeling Denis, don't leave in commented code unless you expect it to come back.

But don't do another commit for this alone; only address if you have to make other changes to the PR.